internal/adapter/handlebars/handlebars_test.go

```diff
@@ -243,6 +243,102 @@ func TestFormatDateHelper(t *testing.T) {
 	testString(t, "{{format-date now 'elapsed'}}", context, "14 years ago")
 }
 
+func TestFormatDateHelperElapsed(t *testing.T) {
+	cases := []struct {
+		elapsed time.Duration
+		want    string
+	}{
+		{
+			elapsed: -12 * time.Minute,
+			want:    "not yet",
+		},
+		{
+			elapsed: 23 * time.Second,
+			want:    "just now",
+		},
+		{
+			elapsed: 1 * time.Minute,
+			want:    "1 minute ago",
+		},
+		{
+			elapsed: 90 * time.Second,
+			want:    "1 minute ago",
+		},
+		{
+			elapsed: 17 * time.Minute,
+			want:    "17 minutes ago",
+		},
+		{
+			elapsed: time.Hour,
+			want:    "1 hour ago",
+		},
+		{
+			elapsed: 3 * time.Hour,
+			want:    "3 hours ago",
+		},
+		{
+			elapsed: 4 * 24 * time.Hour,
+			want:    "4 days ago",
+		},
+		{
+			elapsed: 7 * 24 * time.Hour,
+			want:    "1 week ago",
```

Comment:
Can you add one please that includes "last week", which I think should also work.

Reply:
@zalegrala Can you provide the test case please? FWICT, the result for < 7 days is the number of days, for 7 days is 1 week ago, and for >7 days it is 2 weeks.

I added the case for 2 weeks.

Reply:
```		{
			elapsed: 7 * 24 * time.Hour,
			want:    "last week",
		},
```
I believe this should pass without code changes.

Reply:
That is that specific test case, and it produces `"1 week ago"`.
What am I missing?

Reply:
Oh maybe I misunderstand.  I thought the underlying library that was doing the transformation from string to date is what is being tested here.  Feel free to proceed if the test case I mentioned above doesn't add additional coverage.  The `last week` text is what is being used in some of my templates.

Reply:
it's actually the other way around. The library is calculating the elapsed time and then returning a string.
We're just testing that each possible span of elapsed time, returns the correct string (i.e., elapsed time in natural language)